.gitlab-ci.yml

```diff
@@ -93,10 +93,11 @@ build:rust:linux:release:          &build
     - mkdir -p ./artifacts
     - mv ./target/release/substrate ./artifacts/.
     - echo -n "Substrate version = "
-    - ./artifacts/substrate --version | 
+    - ./artifacts/substrate --version |
       sed -n -r 's/^substrate ([0-9.]+.*-[0-9a-f]{7,13})-.*$/\1/p' |
       tee ./artifacts/VERSION
     - sha256sum ./artifacts/substrate | tee ./artifacts/substrate.sha256
+    - ./scripts/node-template-release.sh ./artifacts/substrate-node-template.tar.gz
```

Comment:
@gabreal could you verify that this is correct? Can I test this somehow?

Reply:
yes, you can do this like that. the files in artifacts will be released to S3 and also be included in the dockerfile.

Reply:
You can test this when you rewrite this file to build it also for pull requests. The publishing although cannot be tested so easily.